.github/workflows/github-cd.yml

```diff
@@ -0,0 +1,44 @@
+name: DocumentGeneration Client NuGet CI
+
+env:
+  PROJPATH: './src/Kmd.Logic.DocumentGeneration.Client'
+  DOTNET_VERSION: '3.1.301'
+
+on:
+  push:
+    branches: [ master ]
+    paths: 
+    - 'src/**'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Get current time
+      uses: 1466587594/get-current-time@v2
+      id: current-time
+      with:
+        format: YYYYMMDD-HHmm
+
+    - name: Setup .NET Core
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: ${{ env.DOTNET_VERSION }}
+    - name: Install dependencies
+      run: dotnet restore '${{ env.PROJPATH }}/Kmd.Logic.DocumentGeneration.Client.csproj'
+
+    - name: Build
```

Comment:
What about build when it is other branches than `master`? Refer to SMS, need to pre-release steps for both `Build` and `Pack` tasks

Reply:
Resolved